app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const { getTopics } = require('./controllers/topics.controllers')
+
+const app = express()
+
+app.use(express.json())
+
+app.get('/api/topics', getTopics)
+
+//custom error handler
+app.use((error, req, res, next) => {
+    if (error.status) {
+        res.status(error.status).send({
+            status: error.status,
+            msg: error.msg
+        })
+    }
+    next(error)
+})
```

Comment:
Similar to the above this shouldnt need to be added yet as it shouldnt be needed for this ticket. (Also see comments on testing for why)